content/ja/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Namespace(名前空間)
 content_type: concept
-weight: 30
+weight: 45
```

Comment:
The original describes `weight: 40`.
Is there a reason you set it to 45?

Reply:
@t-inu 
Thank you for your comment.

The weight was updated in #37479.
And the latest weight is 45.

https://github.com/kubernetes/website/blob/197f18f4afeaac67ec7e33abf6ba34a6ab3f35e2/content/en/docs/concepts/overview/working-with-objects/namespaces.md?plain=1#L6-L8

I added the related issue to this top comment.

Reply:
Should I also update the field-selectors.md fix of #37479 in this PR?

https://github.com/kubernetes/website/pull/37479/files#diff-4c8595cdeee894e2e7e353b64710aee5a99ada80a7c31e3ff2e23f65788315ea

Reply:
Fixed a file in the same directory according to #37479.